sources/sqlserver/toddl.go

```diff
@@ -76,27 +67,25 @@ func toSpannerTypeInternal(conv *internal.Conv, id string, mods []int64) (ddl.Ty
 		return ddl.Type{Name: ddl.Numeric}, nil
 	case "ntext", "text", "xml":
 		return ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, nil
-	case "smalldatetime", "datetimeoffset", "datetime2", "datetime", "timestamp":
+	case "smalldatetime", "datetimeoffset", "datetime2", "datetime":
 		return ddl.Type{Name: ddl.Timestamp}, []internal.SchemaIssue{internal.Timestamp}
 	case "time":
 		return ddl.Type{Name: ddl.String, Len: ddl.MaxLength}, []internal.SchemaIssue{internal.Time}
 	case "varchar", "char", "nvarchar", "nchar":
-		if len(mods) > 0 && mods[0] > 0 && mods[0] <= StringLimit {
+		if len(mods) > 0 && mods[0] > 0 && mods[0] <= stringLimit {
```

Comment:
Suggest adding a comment explaining this custom string and byte limit.

Reply:
Added a comment there

Reply:
I dont see byteLimit used anywhere.

Reply:
Right, that unused constant has been removed recently.